### PR TITLE
feat: Updates for New Weapons and Mutated System!

### PR DESCRIPTION
**Pull request by DrugAddict420**
Hope this will update this fantastic repo!

### CODE REVIEW

File: webapp/src/assets/i18n/en.json [hunk 1]
```diff
@@ -40,7 +42,7 @@
   "ATTRIBUTES_GROUP": {
     "BASE": "Base",
     "ATTRIBUTES": "Attributes",
-    "WEAPON_SKILLS": "Weapons",
+    "WEAPON_SKI= 'LLS": "Weapons",
```
Comment (DrugAddict420) [inline webapp/src/assets/i18n/en.json]: please fix the mistype error should be "WEAPON_SKILLS": "Weapons",

**Comment by DrugAddict420** [comment]
it doesnt let me make another pull request.. please fix the mistype, the rest is good!

**Comment by cbartel** [comment]
> it doesnt let me make another pull request.. please fix the mistype, the rest is good!

its allright, as you commited to your fork, this pull request was updated :)

**Review by DrugAddict420 — COMMENTED** [review]
mistype on line 45 : "WEAPON_SKI= 'LLS": "Weapons", put please: 

"WEAPON_SKILLS": "Weapons",

**Pull request merged**